homeassistant/components/homekit/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 import voluptuous as vol
 
-import homeassistant.components.cover as cover
+from homeassistant.components import cover
```

Comment:
Please correct me if I'm wrong, but as far as I understand the python `import` statement, this does make a difference in memory usage. Whereas with
```py
from homeassistant.components import cover
```
Python will first load **all** components modules and then assign only the `cover` variable to the namespace (all others get disregarded), in
```py
import homeassistant.components.cover as cover
```
it will load only the `cover` package and then assign this one to the namespace.

Reply:
This example suggests that it doesn't load all of them, and that there's no difference in what gets loaded:

```
$ mkdir a
$ echo "print('__init__')" > a/__init__.py
$ echo "print('a')" > a/a.py
$ echo "print('b')" > a/b.py
$ echo "print('c')" > a/c.py
$ python3 -c "import a.b as b"
__init__
b
$ python3 -c "from a import b"
__init__
b
```

Reply:
You're right. Just tested it with a slightly different example: invalid statement in `a/a.py` to look for compiler errors, but it didn't show any. It also didn't generate any `pycache` files for `a/a.py`.

In that case your suggestion does make sense.